src/Models/StageModel.php

```diff
@@ -133,6 +133,30 @@ public function updateAllStages(array $stages) : bool
         }
     }
 
+    public function createOption(string $option, int $stageId) : bool
```

Comment:
Might be worth adding documentation to these functions

Reply:
done